tests/pages/firefox/browsers/compare.py

```diff
@@ -44,7 +31,3 @@ def secondary_download_button(self):
     def browser_menu_list(self):
         el = self.find_element(*self._browser_menu_list_locator)
         return MenuList(self, root=el)
-
-    @property
-    def promo(self):
```

Comment:
It looks like the [test run](https://gitlab.com/mozmeao/www-config/-/jobs/1016570576) failed becuase you still need to remove the test related to this page object

```
E       AttributeError: 'BrowserComparisonPage' object has no attribute 'promo'
```

Reply:
Double checked all files for the instance of this, and this was the only extra one. 🧹